assets/prefabs/items/assemblyProcesses/AssembleCrudeShears.prefab

```diff
@@ -0,0 +1,19 @@
+{
+  "Parent": "CrudeAssemblingBaseProcessPart",
+  "ProcessDefinition": {
+    "processType": "ManualLabor:CrudeAssembling"
+  },
+  "RequirementInput": {
+      "requirements": ["MetalFile"]
+  },
+  "InventoryInput": {
+    "itemCounts": {
+      "ManualLabor:Nugget": 3
+    }
+  },
```

Comment:
I'm still not very happy with this.
What do you think about the following proposal:
2*Rod + 1*Bolt, worked on with a Mallet tool